src/services/services.ts

```diff
@@ -1834,7 +1834,7 @@ namespace ts {
             const fileContents = sourceFile.text;
             const result: TodoComment[] = [];
 
-            if (descriptors.length > 0) {
+            if (descriptors.length > 0 && !isNodeModulesFile(fileName)) {

```

Comment:
The existing function `program.isSourceFileFromExternalLibrary` probably does what you want here.